app/models/questionable/answer.rb

```diff
@@ -1,10 +1,19 @@
 module Questionable
   class Answer < ApplicationRecord
     belongs_to :user
-    belongs_to :assignment
     belongs_to :option
+    belongs_to :parent_question, class_name: 'Questionable::Question', foreign_key: 'question_id',
+               inverse_of: :child_answers
+    belongs_to :assignment
+
+    # TODO: remove `through: :assignment` and `:parent_question` after transition
+    # https://github.com/bespokepost/questionable/pull/19#discussion_r115736332
     has_one :question, through: :assignment
 
+    before_save do
+      self.question_id ||= assignment.try(:question_id)
```

Comment:
this means that answers table will keep the assignment_id column? I assume the intention of this is to avoid introducing breaking changes, right?